src/components/v2v3/V2V3Project/ProjectSafeDashboard/ProjectSafeDashboard.tsx

```diff
@@ -82,21 +111,29 @@ export function ProjectSafeDashboard() {
         </ExternalLink>
       ) : null}
 
-      {isQueuedLoading && <div style={{ marginTop: 20 }}>Loading...</div>}
+      {isLoading && <div style={{ marginTop: 20 }}>Loading...</div>}
 
-      {!isQueuedLoading && (
+      {!isLoading && (
         <div style={{ marginTop: '1.5rem' }}>
           <Space size="large">
-            <Tab
-              name={TAB_NAMES.queued}
-              isSelected={selectedTab === SAFE_TX_QUEUED_KEY}
-              onClick={() => setSelectedTab(SAFE_TX_QUEUED_KEY)}
-            />
-            <Tab
-              name={TAB_NAMES.history}
-              isSelected={selectedTab === SAFE_TX_HISTORY_KEY}
-              onClick={() => setSelectedTab(SAFE_TX_HISTORY_KEY)}
-            />
+            <Link href={`${projectSafeRoute}?tab=queued`}>
+              <a>
+                <Tab
+                  name={TAB_NAMES.queued}
+                  isSelected={selectedTab === SAFE_TX_QUEUED_KEY}
+                  onClick={() => setSelectedTab(SAFE_TX_QUEUED_KEY)}
```

Comment:
**question** now that it's a link, can we remove the onClicks?

Reply:
Yep